### PR TITLE
[4.0] com_templates - remove unused submenu code

### PR DESCRIPTION
**Pull request by brianteeman**
Remove unused references to addSubmenu in various views on com_templates

### CODE REVIEW

File: administrator/components/com_templates/Helper/TemplatesHelper.php [hunk 1]
```diff
@@ -25,27 +25,6 @@
...
-	 * Configure the Linkbar.
-	 *
-	 * @param   string  $vName  The name of the active view.
-	 *
-	 * @return  void
-	 */
-	public static function addSubmenu($vName)
-	{
-		\JHtmlSidebar::addEntry(
-			Text::_('COM_TEMPLATES_SUBMENU_STYLES'),
```
Comment (Quy) [inline administrator/components/com_templates/Helper/TemplatesHelper.php:38]: Remove language string.

Comment (brianteeman) [reply administrator/components/com_templates/Helper/TemplatesHelper.php:38]: The string is used elsewhere

Comment (Quy) [reply administrator/components/com_templates/Helper/TemplatesHelper.php:38]: I did a search for it and not found it elsewhere.

Comment (brianteeman) [reply administrator/components/com_templates/Helper/TemplatesHelper.php:38]: My mistake I misread my search result

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3a319b17ef5391690ae51809af14451b47961dfd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25306">issues.joomla.org/tracker/joomla-cms/25306</a>.</sub>

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 3a319b17ef5391690ae51809af14451b47961dfd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25306">issues.joomla.org/tracker/joomla-cms/25306</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25306">issues.joomla.org/tracker/joomla-cms/25306</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**